.github/bin/check_files.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+function main {
+  expected_files=(metadata.json output error_log results.json output.json)
```

Comment:
Should it also check for `test/hello-world/test/hello_world_test.exs.altered` and `test/hello-world/test/hello_world_test.exs.bkp`?

Reply:
It could but those files are more for our local convenience. IIRC, the test runner checks out a new copy of the solution each time. Runs the tests on that copy, collects the result files, and then deletes the checked out copy.